project/tests/test_actual_time_30_day/_solver_profile_file.json

```diff
@@ -0,0 +1,5 @@
+{
+  "scheduling": { "logging": false, "max_iteration_search_time": 30.0, "num_search_workers": 16 },
```

Comment:
shouldn't use 16 workers to run this test. jsut use a single one. it's better simply don't pass it

Reply:
Can you please tell me what this setting is for? I changed as you say, nothing has changed

Reply:
this setting defines a number of threads to be used for calculation. Better to set it to 0, in this case it will automatically select a proper number.